kong.conf.default

```diff
@@ -1845,3 +1845,7 @@
                                   # behavior and explicitly instructs Kong which
                                   # OpenResty installation to use.
 
+#node_id =                        # Node ID for the Kong node. Every Kong node
+                                  # in a Kong cluster must have a unique and
```

Comment:
```suggestion
                                  # in a Kong cluster must have an unique and
```

Reply:
"a unique" is correct; please see https://preply.com/en/question/which-is-correct-a-unique-or-an-unique-49748